components/NewFooter /index.jsx

```diff
@@ -0,0 +1,28 @@
+import StyleGuide from '../Styleguide'
+import clsx from 'clsx'
+import styles from './styles.module.scss'
+
+const NewFooter = () => {
+    return (
+        //outer and inner explanation?//
```

Comment:
`.outer` and `.inner` are just css classes in `styles/containers.scss` to handle container layout.

`.outer` uses padding left/right to create a 20px gutter in the layout. A gutter (design term) is used so that content doesn't touch the sides of the screen (in this case on mobile screens)

`.inner` set a max width of the content and centers it using margins

In this case just set `Footer` to have
```
<div className='outer'>
  <div className='inner'>
   {/** work goes here */}
  </div>
</div>
```